dags/clone/config/__init__.py

```diff
@@ -1,9 +1,3 @@
-from .paths import DIR_SQL_SCHEMAS, DIR_SQL_VALIDATIONS  # noqa: F401
-from .schemas import (  # noqa: F401
-    SCHEMAS_PREFIX,
-    TABLES,
-    VIEW_NAME_ETAB,
-    VIEW_NAME_UNITE,
-)
+from .model import DIR_SQL_CREATION, DIR_SQL_VALIDATION, CloneConfig  # noqa: F401
```

Comment:
Je suis un peu relou avec ça, mais est-ce que c'est possible d'ajouter la raison du noqa ? à la limite le lien vers ruff ou flake ? 

https://flake8.pycqa.org/en/2.5.5/warnings.html
https://docs.astral.sh/ruff/rules/unused-import/

Ou une mini justification, genre 

```suggestion
from .model import DIR_SQL_CREATION, DIR_SQL_VALIDATION, CloneConfig  # noqa: F401 : unused import
```